support-frontend/conf/routes

```diff
@@ -1,20 +1,20 @@
 # ----- System ----- #
 
-GET /healthcheck                                                   controllers.Application.healthcheck
-GET /error500                                                      lib.ErrorController.error500
-GET /cookies                                                       controllers.DiagnosticsController.cookies
+GET /healthcheck                                                   controllers.Application.healthcheck()
```

Comment:
the auto eta expansion is deprecated as of scala 2.13 (and will work differently in scala 3), so there were a load of warnings.
This also caused some strange compiler errors in the reverse routing in play 2.8, so it needed to be fixed now.